### PR TITLE
Added iOS 11 searchController UINavigationItem

### LINKED ISSUE #376: Search to repo transition has a gross nav bar animation
iOS 11 bug? Maybe we write an interactive transition controller to dynamically crossfade the search bar? Idk, it’s gross tho.

### PR DESCRIPTION
**Pull request by joerocca**
fixes #376

### CODE REVIEW

**Comment by rnystrom** [comment]
Aha! TIL!

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

File: Classes/Repository/RepositoryViewController.swift [hunk 1]
```diff
@@ -62,6 +62,10 @@ NewIssueTableViewControllerDelegate {
         )
         rightItem.accessibilityLabel = NSLocalizedString("More options", comment: "")
         navigationItem.rightBarButtonItem = rightItem
+        
+        if #available(iOS 11, *) {
+            navigationItem.largeTitleDisplayMode = .never
```
[the remark this answers is not in the document]
Comment (rnystrom) [reply Classes/Repository/RepositoryViewController.swift]: We can play around w/ it. I'm personally not a fan of them but might grow on me.

**Comment by rnystrom** [comment]
FYI I reverted this for now, since the iOS 11 API requires a title. Hopefully we can come up w/ something that smoothes the animation w/out requiring the title (unnecessary whitespace).

**Comment by joerocca** [comment]
Sounds good! I will look into smoothing the animation with a custom transition.

**Pull request merged**